Instagram/View Controllers/LoginViewController.m

```diff
@@ -53,6 +53,8 @@ - (IBAction)loginUser:(id)sender {
     }];
 }
 
+
```

Comment:
Let's talk about reviewing your own code before making a commit. This should be an easy thing to avoid.